crdt/src/radixdb.rs

```diff
@@ -301,6 +301,7 @@ pub mod browser {
         }
 
         async fn new0(name: String) -> std::result::Result<BrowserCacheStorage, DomException> {
```

Comment:
new0 is a poor name for a method

Reply:
new_inner? This is a convention from java, to call xxx0 the internals.